tests/test_jsondict.py

```diff
@@ -42,17 +44,17 @@ def test_set_get(self):
         key = 'setget'
         d = self.get_testdata()
         jsd.clear()
-        self.assertFalse(bool(jsd))
-        self.assertEqual(len(jsd), 0)
-        self.assertNotIn(key, jsd)
-        self.assertFalse(key in jsd)
+        assert not bool(jsd)
+        assert len(jsd) == 0
+        assert key not in jsd
+        assert not (key in jsd)
         jsd[key] = d
-        self.assertTrue(bool(jsd))
-        self.assertEqual(len(jsd), 1)
-        self.assertIn(key, jsd)
-        self.assertTrue(key in jsd)
-        self.assertEqual(jsd[key], d)
-        self.assertEqual(jsd.get(key), d)
+        assert bool(jsd)
+        assert len(jsd) == 1
+        assert key in jsd
+        assert key in jsd
```

Comment:
Can we delete this line? This test is identical to the one above.

Reply:
No, this is deliberately testing twice.

Reply:
I need to correct myself, doesn't look like this duplication here is crucial.